android/build.gradle

```diff
@@ -38,5 +38,5 @@ dependencies {
     implementation 'androidx.recyclerview:recyclerview:1.0.0'
     implementation 'androidx.appcompat:appcompat:1.0.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
-    implementation 'com.google.android.material:material:1.0.0-rc01'
+    implementation 'com.google.android.material:material:1.2.1'
```

Comment:
why updating this? I believe this will cause issues because of older zoom SDK used

Reply:
I had to update it to make it work with my app, I cant recall what was the actual error but updating to this version got it working. Ill checkout what happens when I roll it back.

Reply:
Didnt get the issue again so I rolled back the change.